code/modules/research/tg/designs/stock_part_designs.dm

```diff
@@ -578,3 +578,48 @@
 		RND_CATEGORY_STOCK_PARTS + RND_SUBCATEGORY_STOCK_PARTS_5
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE | DEPARTMENT_BITFLAG_ENGINEERING
+
+//Super SMES Coils
+/datum/design_techweb/smes_super_capacity_ultra
+	name = "Superconductive Capacity Coil, Ultra"
+	build_type = PROTOLATHE
+	id = "smes_cap_ultra"
+	materials = list(MAT_STEEL = 1700, MAT_GLASS = 550, MAT_DURASTEEL = 230, MAT_MORPHIUM = 320, MAT_METALHYDROGEN = 600, MAT_URANIUM = 60, MAT_VERDANTIUM = 150, MAT_PHORON = 900)
```

Comment:
For materials, consider using whole sheets or near whole sheet amounts since I can't see a reason to use more than a few of these at a time (except pure blissful overkill in a compact package). Each part tier reduces the cost by an extra 20%. SMES coils also won't be used that heavily, maybe only six at most a shift if someone is really doin' wacky projects. An example cost for the ultras could be something like:

`materials = list(MAT_STEEL = 10000, MAT_GLASS = 6000, MAT_GOLD = 5000, MAT_SILVER = 5000, MAT_DURASTEEL = 2000,  MAT_METALHYDROGEN = 2000, MAT_VERDANTIUM = 2000)`

For the hypers, I would increase mat costs by 50% and also add a sheet of morphium to that on top of it. But that's just me perhaps.